site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the resources in a file/directory
+---
+
+<!--mdtogo:Short
+    Print the resources in a file/directory
+-->
+
+Cat prints the resources in a file/directory as yaml to stdout.
+
+Cat is useful for printing the resources in a file/directory which might
+contain other non-resource files.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt pkg cat [FILE | DIR]
+
+DIR:
+  Path to a directory with resources. Defaults to the current working directory.
+
+FILE:
+  Path to a resource file.
+```
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+```
+# print resource from a file
+kpt pkg cat path/to/deployment.yaml
+```
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+```
+
+```
+# print resources from current directory
+kpt pkg cat
+```
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+---
+apiVersion: apps/v1
+kind: Service
+metadata:
+  name: nginx-service
+```
+
+<!--mdtogo-->
+
+#### Flags
+
+```sh
```

Comment:
Isn't docs for flags auto-generated from cobra configuration?

Reply:
Yes. This should be cleaned up in cobra as well in kustomize repo in a different PR. https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/cat.go#L35